File: project-c-cosminpac04/motorcycleApp/Models/Team.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace motorcycleApp.Models
{
    public class Team : Identifiable<int>
    {
        [Key]
        public int ID { get; set; }
        
        public string TeamName { get; set; }
        
        public List<Participant> TeamMembers { get; set; } = new List<Participant>();

		public Team(){}

        public Team(int id, string teamName)
        {
            ID = id;
            TeamName = teamName;
            TeamMembers = new List<Participant>();
        }
    }
}

File: project-java-cosminpac04/motorcycleContest/src/main/java/motorcycle/model/Team.java
package motorcycle.model;
import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;


public class Team  implements Identifiable<Integer>, Serializable {
    private int ID;
    private String teamName;
    private List<Participant> teamMembers;

    //constructor
    public Team(int ID,String teamName){
        this.ID=ID;
        this.teamName = teamName;
        this.teamMembers = new ArrayList<>();
    }

    //getters
    public String getTeamName() {
        return teamName;
    }
    public List<Participant> getTeamMembers() {
        return teamMembers;
    }
    @Override
    public Integer getID() {
        return ID;
    }

    //setters
    public void setTeamMembers(List<Participant> teamMembers) {
        this.teamMembers = teamMembers;
    }
    public void setTeamName(String teamName) {
        this.teamName = teamName;
    }

    @Override
    public void setID(Integer ID) {
        this.ID = ID;
    }
}
